Tensile/KernelWriterSource.py

```diff
@@ -1046,7 +1046,7 @@ def allocateResources(self, kernel):
     # allocate local memory
     kStr += self.endLine
     kStr += "  /* allocate local memory */" + self.endLine
-    kStr += "  %sDATA_TYPE localMemory[LDS_NUM_ELEMENTS];%s" \
+    kStr += "  %sCOMPUTE_DATA_TYPE localMemory[LDS_NUM_ELEMENTS];%s" \
```

Comment:
Would you please keep LDS allocation size unchanged for LocalSplitU = 1 case?
LDS allocation size will be doubled in HHS case with LocalSplitU = 1 if we use computeDataType here.

I suggest to modify ldsNumElementsReduction part.
Something like
  ldsNumElementsReduction *= sizeof(computeDataType) / sizeof(DataType)

Reply:
pre_checkin/igemm_hpa_hip_nn.yaml seems to be failed due to this change.

b"error: local memory (99712) exceeds limit (65536) in function 'Cijk_Ailk_Bljk_I8II_BH_MT128x128x32_SN_GRVW8_GSU1_K1_LPA3_LPB3_LRVW8_TT8_8_VW8_WG16_16_1'\nerror: local memory (99712) exceeds limit (65536) in function 'Cijk_Ailk_Bljk_I8II_BH_MT128x128x32_SN_GRVW8_GSU3_K1_LPA3_LPB3_LRVW8_TT8_8_VW8_WG16_16_1'\n2 errors generated when compiling for gfx1010.\n"

Reply:
Updated. @nakajee

Reply:
@nakajee Updated.